src/AWSCore.jl

```diff
@@ -555,7 +555,7 @@ function do_request(r::AWSRequest; return_headers=false)
     end
 
     # Return raw data by default...
-    return (return_headers ? (response.body, nothing) : response.body)
+    return (return_headers ? (response.body, Dict(response.headers)) : response.body)
```

Comment:
```suggestion
    return (return_headers ? (response.body, response.headers) : response.body)
```
It says "return raw data by default" and this is what returning raw data above does (L505)